.env.example

```
@@ -83,3 +83,6 @@ NEXT_PUBLIC_NETLIFY_URL=https://your-netlify-project-url.netlify.app
 
 # ENVIRONMENT
 NODE_ENV=development
+
+# DEV TUNNEL (Optional - for ngrok or similar local tunnel workflows)
+NEXT_PUBLIC_DEV_ORIGIN=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)